lib/rucio/core/replica.py

```diff
@@ -804,16 +804,19 @@ def _build_list_replicas_pfn(
         sign_urls: bool,
         signature_lifetime: int,
         client_location: "Dict[str, Any]",
+        *,
```

Comment:
what's this needed for?

Reply:
the asterisk signifies the end of positional arguments and states that every argument that follows is a keyword argument. decorators such as `read_session` or `stream_session` attach to this. since `_build_list_replicas_pfn` is not decorated, the asterisk is not necessary.

I added the asterisk for consistency with functions such as [`_list_files_wo_replicas`](https://github.com/rucio/rucio/blob/6f07c57c0c588c4e6ea03b40e977877be2a59b5b/lib/rucio/core/replica.py#L680).